MIDAS/src/hardware/Continuity.cpp

```diff
@@ -1,17 +1,23 @@
 #include "sensors.h"
+#include "ads7138-q1.h"
 
-// #include sensor library
-
-// global static instance of the sensor
+#include <hal.h>
 
+ADS7138_1Q ads7138_1q;
 
 ErrorCode ContinuitySensor::init() {
     // do whatever steps to initialize the sensor
     // if it errors, return the relevant error code
+    ads7138_1q.init();
```

Comment:
if this function returns true, return no error, if it returns false, we want to return a new error code that you'll have to make in error.h